go.mod

```diff
@@ -4,6 +4,7 @@ go 1.16
 
 require (
 	github.com/antlr/antlr4/runtime/Go/antlr v0.0.0-20210521073959-f0d4d129b7f1
+	github.com/bluele/gcache v0.0.2
```

Comment:
Is there any reason to use this instead of https://github.com/hashicorp/golang-lru? We use the latter in `neofs-node` and `neo-go` and it looks more mature.